program/tests/interface.rs

```diff
@@ -1123,3 +1124,89 @@ fn test_no_use_dealloc() {
         }
     }
 }
+
+// this prints ballpark compute unit costs suitable for insertion in README.md
+// run with `cargo test --test interface show_compute_usage -- --nocapture --ignored`
+#[test]
+#[ignore]
+fn show_compute_usage() {
```

Comment:
Wonder if we'll want to break things out into a benchmarking crate one day. Not sure, but think this is cool for now 👍